tests/test_jobs.py

```diff
@@ -214,11 +217,11 @@ def test_align_cram(self):
         self.assertEqual(sid, _read_file(cram_details_paths['sample_name']))
         cram_details = {k: _read_file(v) for k, v in cram_details_paths.items()}
         print(cram_details_paths)
-        self.assertAlmostEqual(223007, int(cram_details['reads_num']), delta=50)
+        self.assertAlmostEqual(223007, int(cram_details['reads_num']), delta=100)
```

Comment:
I love the name of this method

Reply:
Haha. We also need such metric for the CPG: "pipeline almost worked", "seqr load is almost done"